docker-build.sh

```diff
@@ -5,6 +5,123 @@
 set -o errexit
 set -o xtrace
 
+ARCHIVE_ADDR=https://archive.ubuntu.com/ubuntu/
+PORTS_ADDR=https://ports.ubuntu.com/
+
+# Prepare HWA headers, libs and drivers for x86_64-linux-gnu
+prepare_hwa_amd64() {
+    # Download and install the nvidia headers from deb-multimedia
+    pushd ${SOURCE_DIR}
+    git clone --depth=1 https://git.videolan.org/git/ffmpeg/nv-codec-headers.git
+    pushd nv-codec-headers
+    make
+    make install
+    popd
+    popd
+
+    # Download and setup AMD AMF headers from AMD official github repo
+    # https://www.ffmpeg.org/general.html#AMD-AMF_002fVCE
+    svn checkout https://github.com/GPUOpen-LibrariesAndSDKs/AMF/trunk/amf/public/include
+    pushd include
+    mkdir -p /usr/include/AMF && mv * /usr/include/AMF
+    popd
+
+    # Download and install libva
+    pushd ${SOURCE_DIR}
+    git clone -b v2.6-branch --depth=1 https://github.com/intel/libva
+    pushd libva
+    sed -i 's|getenv("LIBVA_DRIVERS_PATH")|"/usr/lib/jellyfin-ffmpeg/dri"|g' va/va.c
+    sed -i 's|getenv("LIBVA_DRIVER_NAME")|NULL|g' va/va.c
+    ./autogen.sh
+    ./configure --prefix=/usr
```

Comment:
`prefix` value doesn't look correct, I think it should be `/usr/lib/jellyfin-ffmpeg`

Reply:
You got it wrong. This `/usr` is the path when building `ffmpeg` in docker, in order to allow `lib` and `include` to be correctly recognized by `intel-vaapi-driver` and `ffmpeg`, so that they can start building. Once the build is complete, only the files specified in `debian/ jellyfin-ffmpeg.install` will be included in the deb package.

These dockers are all disposable, and can be cleaned up every time the build is completed.

Same about `prefix` in other parts.

Reply:
No, I got it right. :)
The fact that stuff works _now_ after you move all files built in docker to some other location is a fortunate happenstance (i.e. it's not guaranteed to happen). You need to make additional actions to ensure your binaries could be shuffled like that, because sometimes the prefix is baked inside the library as part of some path, and then everything break.

So what I'm saying is not that _you_ did it completely wrong (it was already a bit wrong before this PR 😄), but rather I would like to see a cleaner approach. That file was based off `debian/control` which installed the results into `/usr`, so for it the prefix was right. But we aren't installing into `/usr`! I'm only saying `prefix` should really match with where we're installing, otherwise some subtle and hard-to-understand bugs may arise.